packages/vow/src/watch.js

```diff
@@ -1,21 +1,21 @@
 // @ts-check
+import { PromiseWatcherI } from '@agoric/base-zone';
 import { getVowPayload, basicE } from './vow-utils.js';
-import { PromiseWatcherI, watchPromiseShim } from './watch-promise.js';
 
 const { apply } = Reflect;
 
 /**
- * @param {typeof watchPromiseShim} watchPromise
+ * @param {import('@agoric/base-zone').Zone} zone
  */
 const makeWatchNextStep =
-  watchPromise =>
+  zone =>
   /**
    * If the specimen is a vow, obtain a fresh shortened promise from it,
    * otherwise coerce the non-vow specimen to a promise.  Then, associate a
    * (usually durable) watcher object with the promise.
    *
    * @param {any} specimen
-   * @param {import('./watch-promise.js').PromiseWatcher} promiseWatcher
+   * @param {import('@agoric/base-zone').PromiseWatcher} promiseWatcher
```

Comment:
Now that we have `@import`, we should stop writing `{import('...')...}`. Likewise for other occurrences.

```suggestion
  /** @import {PromiseWatcher} from '@agoric/base-zone' */
   ...
   * @param {PromiseWatcher} promiseWatcher
```

Reply:
I can do that in a future refactor PR (soon to be revealed), but I'd prefer not to include it in this one.

Reply:
#9329 is the refactor.

Reply:
That's fine. Resolving.